deploy/sfsturbo-csi-plugin/kubernetes/csi-sfsturbo-controller.yaml

```diff
@@ -24,23 +24,28 @@ spec:
           effect: "NoSchedule"
       containers:
         - name: csi-provisioner
-          image: quay.io/k8scsi/csi-provisioner:v1.4.0
+          image: k8s.gcr.io/sig-storage/csi-provisioner:v3.1.0
           args:
             - "-v=5"
             - "--csi-address=$(ADDRESS)"
-            - "--enable-leader-election"
-            - "--leader-election-type=leases"
+            - "--timeout=10m"
+            - "--default-fstype=ext4"
```

Comment:
Does SFS Turbo use fstype?

Reply:
this is a useless arg. csi-provisioner do not support this arg.
"--default-fstype=ext4" need to be deleted